src/js/stores/MesosStateStore.js

```diff
@@ -128,20 +128,15 @@ var MesosStateStore = Store.createStore({
     let services = this.get('lastMesosState').frameworks;
     let foundTask = null;
 
-    for (let serviceIndex in services) {
-      let service = services[serviceIndex];
+    services.some(function (service) {
```

Comment:
Yeah, we were doing `for (let serviceIndex in services)` which is less-than-ideal as services (lastMesosState. frameworks) is an array. So we should have fix that anyway.